### PR TITLE
Added back tournaments

### PR DESCRIPTION
**Pull request by DieuwerH**
Added the tournaments, removed sponsors and schedule where they existed and added PUBG.

Need to add rules to the PUGB tournament.

### CODE REVIEW

File: scripts/official-tournaments.json [hunk 1]
```diff
@@ -1,98 +1,107 @@
...
...on time for their matches, if they are late they will suffer a penalty. This can be either losing a game after 20 minutes, or the entire match after 30 minutes as to not delay the entire tournament. A player is required to inform the tournament staff if they are late for a match, including a valid reason. If not they will be dropped from the tournament.\n4. One 5 minute break is allowed during a match.\n5. Coaching is not allowed during a match.\n\n## 5. Tournament Eligibility ##\n\n1. Participants which have a conflict of interest may be excluded from tournaments.\n2. The current organizing committee (LANcie) have the final say in tournament eligibility\n\n## 6. Penalties ##\n\nDepending on the situation a player could be penalized in any of the following ways:\n\n1. Score penalization (eg. starting their next match 0-1 instead of 0-0)\n2. Penalized for one match (eg. forfeiting a match)\n3. Disqualification from the tournament entirely\n\n## 7. Drop out ##\n\nIf a player decides to drop out of the tournament they are required to contact the administrators and sign a drop out sheet for confirmation. Once a player has dropped out of the tournament they will not be able to rejoin.",
```
Comment (Jipje) [inline scripts/official-tournaments.json]: Is at an idea to create a sort of Area Fifty LAN code of conduct? The Behaviour and Conduct section is relevant to all tournaments is it not?

File: scripts/official-tournaments.json [hunk 2]
```diff
@@ -1,98 +1,107 @@
...
...ave a conflict of interest may be excluded from tournaments.\n2. The current organizing committee (LANcie) have the final say in tournament eligibility\n\n## 6. Penalties\nDepending on the situation a player may be penalized in any of the following ways:\n1. Point reduction\n2. Penalized a match (eg. forfeiting a match)\n3. Disqualification from the tournament entirely.\n\n## 7. Drop out ##\nIf a player decides to drop out of the tournament they are required to contact the admin team and sign a drop out sheet for confirmation. Once a player has dropped out of the tournament they will not be able to rejoin.\n## 8. Deck reveal ##\nAfter the swiss rounds have finished, all advancing players decklists will be sent out to the top 8 players advancing to the single elimination bracket.\n## 9. Deck Tracking software ##\nPlayers are allowed to bring a pen and a blank piece of paper for notes during matches. Any kind of Decktracking software is not allowed in this tournament.\n## 10 System requirements ##\nAll tournament devices must fulfill the minimum system requirements for Hearthstone: Heroes of Warcraft.\n To view all the system requirements, please go to the Blizzard Support website.",
```
Comment (Jipje) [inline scripts/official-tournaments.json]: I would make header 2 with bullet points.
As well as add to header 2 that all decks are to be Standard decks, or Wild decks, whichever tournment we want. I feel like we should take only Standard decks.

File: scripts/official-tournaments.json [hunk 3]
```diff
@@ -1,98 +1,107 @@
...
....\n - Recordings and/or screenshots from each round are required\n\n## Map Pick/Veto ##\nUnless otherwise specified, the map pick/veto rules are as follows:\n - The highest seeded team will be chosen to start the pick &amp; ban phase\n - Contestants ban maps until number of maps needed.\n - Maps are played in the order they were picked.\n\n## Technical Issues: ##\n - In the case that a player(s) disconnects, the team is allowed to continue to play the match at a disadvantage.\n - If a player fails to show up on time the team can decide to wait (upto 10 minutes) or play with a player less.\n - If a team chooses to wait and after 10 min the missing player still hasn’t shown up the team must play with a player less.\n - If a missing player shows up in the middle of a round he/she is not allowed to join and must wait until the next round.\n - If a match can’t be played because of multiple players missing, the administrator can choose to wait if there is enough time or penalize the team with a lost round or a complete loss of the match.\n - If the administrator chooses to wait, then after 10 min the faulty team is penalized with a lost round and after 20 min complete disqualification.",
```
Comment (Jipje) [inline scripts/official-tournaments.json]: Maybe what we specify here at the header Technical Issues, should be said everywhere. For example in the "Area Fifty LAN Code of Conduct" Especially the part about how being late to a tournament could lead to disqualification. That is something we might be able to use well to ensure the schedule.

File: scripts/official-tournaments.json [hunk 4]
```diff
@@ -1,98 +1,107 @@
...
...s he or she likes. A selfie with the time MUST be provided to one of the administrators as proof.\n## 3. Behavior and Conduct ##\n\n1. If a participant commits - or threatens to commit - violent acts against any other player in the Tournament, any admin, or any spectators of the Tournament, he or she may be disqualified from the Tournament.\n2. If a participant behaves in any unsportsmanlike way it is possible that he or she receives a penalty, to the sole discretion of the administrators.\n3. It is a players responsibility to be on time for their matches, if they are late they will suffer a penalty. This can be either losing a game after 20 minutes, or the entire match after 30 minutes as to not delay the entire tournament. A player is required to inform the tournament staff if they are late for a match, including a valid reason. If not they will be dropped from the tournament.\n4. One 5 minute break is allowed during a match.\n5. Coaching is not allowed during a match.\n\n## 4. Tournament Eligibility ##\n\n1. Participants which have a conflict of interest may be excluded from tournaments.\n2. The current organizing committee (LANcie) have the final say in tournament eligibility",
```
Comment (Jipje) [inline scripts/official-tournaments.json]: Looks good.

File: scripts/official-tournaments.json [hunk 5]
```diff
@@ -1,98 +1,107 @@
...
+    },
+    "prizes": [
+      "To be announced"
+    ]
+  },
+  {
+    "title": "PUBG",
+    "subtitle": "FFA",
+    "headerTitle": "PlayerUnknown's Battlegrounds",
+    "description": "",
```
Comment (Jipje) [inline scripts/official-tournaments.json]: Can we take the relevant rules from the [following?](https://play.eslgaming.com/playerunknownsbattlegrounds/europe/pubg/open/gamescom-invitational-2017-registration/rules/global#!#wdh_1_4-Game202f20client)

Comment (martijnjanssen) [reply scripts/official-tournaments.json]: I think a lot can be used from that, but not everything will be applicable.

File: scripts/official-tournaments.json (hunk 1 above)
Comment (Fastjur) [reply scripts/official-tournaments.json]: That would make sense yes!

Comment (Fastjur) [reply scripts/official-tournaments.json]: Done

File: scripts/official-tournaments.json (hunk 3 above)
Comment (Fastjur) [reply scripts/official-tournaments.json]: This is done. We should possibly alter the terms of service a bit if need me

File: index.html [hunk 6]
```diff
@@ -342,4 +342,4 @@
     })();
   </script>
 </body>
-</html>
\ No newline at end of file
+</html>
```
Comment (Fastjur) [inline index.html]: hmm

File: src/lancie-my-area/lancie-terms-of-service.html [hunk 7]
```diff
@@ -16,8 +16,9 @@
       .dialog-content {
         flex: 1;
         height: 0%; /* Ayy Lmao */
+        overflow-y:scroll;
```
Comment (Fastjur) [inline src/lancie-my-area/lancie-terms-of-service.html]: I'm sorry, this doesn't work on my machine

Comment (Fastjur) [reply src/lancie-my-area/lancie-terms-of-service.html]: Nvm was a caching issue

File: index.html (hunk 6 above)
Comment (martijnjanssen) [reply index.html]: The newline is stripped since this is probably edited in windows.

File: src/lancie-activity/lancie-activity-content.html [hunk 8]
```diff
@@ -14,11 +15,13 @@
       }
       .description {
         text-align: justify;
+        padding-right: 16px;
       }
     </style>
 
     <div class="content">
-      <marked-element markdown="{{data.description}}"></marked-element>
+      <marked-element markdown="{{data.description}}" class="description"></marked-element>
```
Comment (martijnjanssen) [inline src/lancie-activity/lancie-activity-content.html]: Data binding can be one-way.

**Comment by martijnjanssen** [comment]
Another thing, lets change the overflow property of the content in the activity dialog from `overflow: scroll;` to `overflow: auto;`, this results in the scrollbar only being in the div if the content is actually scrollable.

**Pull request merged**